cf_xarray/accessor.py

```diff
@@ -1340,12 +1340,36 @@ def rename_like(
         theirkeys = other.cf.keys()
 
         good_keys = ourkeys & theirkeys
-        renamer = {}
+        keydict = {}
         for key in good_keys:
             ours = _single(_get_all)(self._obj, key)[0]
```

Comment:
Should we catch the error and just skip+warn?

Reply:
yes that sounds right